data-manager/src/main/java/org/orbisgis/datamanager/h2gis/H2GIS.java

```diff
@@ -237,4 +237,13 @@ public ISpatialTable getSpatialTable(String tableName) {
             return null;
         }
     }
+
+    @Override
+    public boolean hasTable(String tableName) {
+        try {
+            return JDBCUtilities.tableExists(connectionWrapper,tableName);
+        } catch (SQLException ex) {
+            return false;
```

Comment:
The exception should be at least logged.

Reply:
idem